pkg/controller/factory/factory.go

```diff
@@ -54,6 +61,12 @@ func (f *Factory) WithInformers(informers ...Informer) *Factory {
 	return f
 }
 
+// WithPostRunHooks allows to register functions that will run asynchronously after the controller is started via Run command.
+func (f *Factory) WithPostRunHooks(hooks ...PostRunHook) *Factory {
```

Comment:
see: https://github.com/openshift/library-go/pull/731/commits/80202dc074c17a2981110c7ce0b7fc706a8fde8d#diff-2261944c4e72b8534466fe56d0a53f41R119 for example.